distgen/project.py

```diff
@@ -6,15 +6,24 @@ class AbstractProject():
     maintainer = "unknown <unknown@unknown.com>"
 
 
-    def inst_init(self, spec, template, config):
+    def inst_init(self, specfiles, template, sysconfig):
         """
-        Returns dict with data to be used in specification file.
+        Executed before the project.py/spec files/template is loaded and
+        before all the dynamic stuff and specification is calculated.
+        Now is still time to dynamically change the list of specfiles or
+        adjust the system configuration.  Any object variable (defined
+        here) will be available in jinja template under 'project.*'
```

Comment:
Does this also hold for final templating phase? (I can see you changed the `_eval_node` function which confuses me)

Reply:
The `_eval_node` change was solely to simplify api of this callback;  and to avoid additional `{{ init.* }}` namespace in templates... since for the same purpose might be used `{{ project.* }}`.